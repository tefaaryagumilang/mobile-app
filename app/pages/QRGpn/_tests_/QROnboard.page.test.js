import React from 'react';
import renderer from 'react-test-renderer';
import QROnboard from '../QROnboard.page';
import {Provider} from 'react-redux';
import {createStore} from 'redux';
import {reduxForm} from 'redux-form';

jest.mock('../QROnboard.page');
const store = createStore(() => ({}));
const DecoratedForm = reduxForm({form: 'QRForm'})(QROnboard);

describe('Home: QROnboard page', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Provider store={store}><DecoratedForm/></Provider>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
