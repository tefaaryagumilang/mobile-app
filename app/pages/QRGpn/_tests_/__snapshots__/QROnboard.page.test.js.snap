// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home: QROnboard page renders correctly 1`] = `null`;
